FIAT/bernstein.py

```diff
@@ -53,6 +55,9 @@ def __init__(self, ref_el, degree):
         dual = BernsteinDualSet(ref_el, degree)
         k = 0  # 0-form
         super().__init__(ref_el, dual, degree, k)
+        pts = make_lattice(ref_el.vertices, degree)
```

Comment:
```suggestion
        pts = make_lattice(ref_el.vertices, degree, variant="gll")
```

Reply:
For the TL/DR:  I think Bernstein is the only use case we currently have, so I'm hesitant to spend too much work sniffing out for the nodes being `None` and then overwriting the dual if it is.

1. Hard to automate unless you're just on a P^k space so know what dimension space you have/how many points.
2.  FWIW, serendipity already does something similar to this to provide the dual set for the manually-specified basis functions.  But we had to be careful to specify the points as it isn't just `make_lattice`.  
3. Other use case would be Hermite or Argyris, if you wanted to get a dual that doesn't refer to derivatives (so you could interpolate by some kind of finite differences).  Seems like a different issue to try to resolve/engineer.